source/sonata_simulation.rst

```diff
@@ -85,6 +85,7 @@ Parameters defining global simulation settings. As NEURON is the engine used for
    ionchannel_seed                 integer    Optional    A non-negative integer used for seeding stochastic ion channels, default is 0.
    minis_seed                      integer    Optional    A non-negative integer used for seeding the Poisson processes that drive the minis, default is 0.
    synapse_seed                    integer    Optional    A non-negative integer used for seeding stochastic synapses, default is 0.
+   electrodes_file                 text       Optional    Filename that contains the weights for the LFP calculation.
```

Comment:
has this file been fully documented?

Reply:
@joseph-tharayil Started that in another PR, so I will close this one and add the field there.
https://github.com/BlueBrain/sonata-extension/pull/11